doc/sphinx/practical-tools/coqide.rst

```diff
@@ -209,51 +228,48 @@ the *View* menu, or using the shortcut ``F2``.
 You can also do queries by selecting some text, then choosing an
 item from the *Queries* menu. The response will appear in the message panel.
 The image above shows the result after selecting
-``Nat.mul`` in the bottom line of the script panel, then choosing ``Print``
-from the ``Queries`` menu.
-
-.. todo: should names of menus be *Menu* or `Menu` or ??  not consistent
-
+``Nat.mul`` in the bottom line of the script panel, then choosing *Print*
+from the *Queries* menu.
 
 Compilation
 -----------
 
-The `Compile` menu offers direct commands to:
+The *Compile* menu offers direct commands to:
 
-+ compile the current buffer
-+ run a compilation using `make`
-+ go to the next compilation error and
++ compile the current buffer;
```

Comment:
FYI Semicolons or commas between bullet items is optional in English except for formal writing such as academic papers or government documents.